src/Employer.Web/Services/GetAssociatedEmployerAccountsService.cs

```diff
@@ -1,10 +1,49 @@
-﻿namespace Employer.Web.Services
+﻿using Microsoft.Extensions.Logging;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.EAS.Account.Api.Types;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Employer.Web.Services
 {
     public class GetAssociatedEmployerAccountsService : IGetAssociatedEmployerAccountsService
     {
-        public string[] GetAssociatedAccounts(string userId)
+        private readonly ILogger<GetAssociatedEmployerAccountsService> _logger;
+        private readonly IAccountApiClient _accountApiClient;
+
+        public GetAssociatedEmployerAccountsService(ILogger<GetAssociatedEmployerAccountsService> logger, IAccountApiClient accountApiClient)
         {
-            return new string[] { "abc", "xyz" };
+            _logger = logger;
+            _accountApiClient = accountApiClient;
+        }
+
+        public async Task<string[]> GetAssociatedAccounts(string userId)
```

Comment:
I've appended 'Async' to my async functions so GetAssociatedAccountsAsync

Reply:
👍